src/main/java/org/sqlite/jdbc3/JDBC3ResultSet.java

```diff
@@ -311,46 +312,21 @@ public Date getDate(int col) throws SQLException {
      * @see java.sql.ResultSet#getDate(int, java.util.Calendar)
      */
     public Date getDate(int col, Calendar cal) throws SQLException {
```

Comment:
It seems like this moves the driver away from the JDBC spec by ignoring the user supplied Calendar/Timezone information.